.landscaper/logging-stack/resources-sap-btp-service-operator.yaml

```diff
@@ -12,10 +12,10 @@ input:
 type: helm.io/chart
 name: sap-btp-service-operator-chart
 relation: external
-version: v0.5.4
+version: v0.5.6
```

Comment:
Is this here updated by accident? Can we move this dependency update to a seperate PR?